views/pages/resources/important-links.pug

```diff
@@ -0,0 +1,16 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+    link(href="/assets/css/pages/resources/important-links.css" rel="stylesheet")
+
+block body 
+    div(class="content")
+        section 
+            h1 #{data.title}
+            ol(class="list")
+                each link in data.links
+                    li(class="list-item")
+                        div(class="list-item-content")         
+                            a(href=link.link target="_blank" class="list-item-link")
+                                img(class="list-item-icon" src=link.icon)
+                                h3(class="list-item-title") #{link.title}
```

Comment:
We should be avoiding using `h3` unless it's for a title - it will make it easier for screenreaders used by the visually-impaired to navigate the page